code/CSS Grid Assignment/style.css

```diff
@@ -0,0 +1,38 @@
+#wrapper {
+    font-size: 2em;
+    gap: 1rem;
+    /* Add new properties here */
+  }
+  
+  /* Add new selectors and new CSS here */
+  #wrapper{
+    display:grid;
+    grid-template-columns:4em 10em 1/2fr;
+    grid-template-rows: 
+}
+  #header{
+    grid-column-start:1;
+    grid-column-end:4;
+  }
+
+  #left{
+    width:4em;

```

Comment:
In this case, if you want the width to take up the full column, it's not necessary (and actually causes problems) to define the width separately from the grid definition. Best to let the grid handle it.